server/pbench/bin/pbench-rsync-satellite

```diff
@@ -140,9 +136,8 @@ for host in $hosts ;do
     if [ -s $md5list ] ;then
         md5sum -c $(cat $md5list) | sed "s/^/$host: /" > $logdir/$host/md5-checks.log
         # delete all that check out
-        for x in $(sed -n '/OK$/s/: OK//p' $logdir/$host/md5-checks.log) ;do
-            echo $x $x.md5
-        done | ssh $remotehost "cd $remotearchive/$host; xargs rm" > $logdir/$host/rm.log 2>&1
+        grep 'OK' $logdir/$host/md5-checks.log |
+            ssh $remotehost "cd $remotearchive/$host; /opt/pbench-server/bin/pbench-satellite-cleanup" > $logdir/$host/rm.log 2>&1
```

Comment:
If the ssh fails, what happens?

It might be worth our time to save the output of the `grep` to a file, if the file is not empty, then `scp` that to the remote host (checking that the scp worked), and then `ssh` to the remote host to run the `pbench-satellite-cleanup` command, checking that it succeeded.